src/components/ProjectForm.js

```diff
@@ -27,6 +27,10 @@ export const ProjectForm = props => {
         label='Description'
         placeholder='Description'
       />
+      <Field name='slack_channel_name' component={TextAreaField}
+        label='slack channel name'
```

Comment:
I wonder if it's not better to capitalize the label?

Reply:
@mattwr18 fixed